assets/src/components/parts/janus-slider/Slider.tsx

```diff
@@ -285,16 +291,21 @@ const Slider: React.FC<PartComponentProps<SliderModel>> = (props) => {
               className={` slider `}
               id={internalId}
               onChange={handleSliderChange}
+              list={showTicks ? `lst${internalId}` : ''}
             />
+            {showTicks && (
+              <datalist id={`lst${internalId}`}>
```

Comment:
I tried this locally and didn't see anything. is there supposed to be some CSS changes to go with it? also, not sure about what appears to be hard coded "ticks" shouldn't they be reflected dynamically based on the increment value and the total? (so if increment is 1 there would be 100 tick marks if the max is 100)

Reply:
yeah... this part got checked-in by mistake.... actually this is in progress.. other two issues mentioned in the ticket are resolved in this PR.